compiler-core/src/parse/tests.rs

```diff
@@ -1121,3 +1121,22 @@ fn newline_tokens() {
         ]
     );
 }
+
+#[test]
+fn const_string_concat() {
+    assert_parse_module!(
+        "
+const foo = \"foo\"
+const bar = foo <> \"bar\"
+"
+    );
+}
+
+#[test]
+fn const_string_concat_naked_right() {
+    assert_module_error!(
+        "
+const foo = \"foo\" <>
```

Comment:
Remove the foobar from this file please 🙏

Reply:
oops sorry missed this one